test/unit/model-class-typings.test.ts

```diff
@@ -44,7 +44,7 @@ describe("Model Class static attributes typings", () => {
           const endpoint: string = RootClass.endpoint
           const jwt: string | undefined = RootClass.jwt
           const jwtLocalStorage: string | false = RootClass.jwtLocalStorage
-          const camelizeKeys: boolean = RootClass.camelizeKeys
+          const keyCase: KeyCase = RootClass.keyCase
```

Comment:
I donot know about this test. It should fail in my opinion but does not. also if I change another attributes type to something diffrent. 

Since this is my first time writing typescript I appreciate some help here.

Reply:
Not sure I'm following your question.  This test is essentially serving as a smoke test that the type of a particular static property is what you'd expect.  If you look at [the comment at the top](https://github.com/jannishuebl/jsorm/blob/0070e40f1d186e0d9c7e09a3d3e9f9275d4f4c94/test/unit/model-class-typings.test.ts#L13-L23), the test won't actually ever fail, it will just be a type error in your editor or at compilation time.  The current state of `yarn run test` doesn't actually throw type errors, so it's more of a user assistance test than anything.